app/web/features/landing/LandingPage.tsx

```diff
@@ -204,33 +212,66 @@ export default function LandingPage() {
       <StyledSection>
         <StyledContent>
           <StyledIntroduction>
-            <Typography
-              variant="h1"
-              component="h1"
-              sx={{
-                [theme.breakpoints.up("md")]: {
-                  fontSize: "2rem",
-                  lineHeight: "1.15",
-                  textAlign: "left",
-                },
-              }}
-            >
-              {t("landing:introduction_title")}
-            </Typography>
-            <Typography
-              variant="h2"
-              component="span"
-              sx={{
-                [theme.breakpoints.up("md")]: {
-                  display: "inline-block",
-                  marginTop: theme.spacing(4),
-                  position: "relative",
-                },
-              }}
+            <StyledIntroductionText>
+              <Typography
+                variant="h1"
+                component="h1"
+                sx={{
+                  [theme.breakpoints.up("md")]: {
+                    fontSize: "2rem",
+                    lineHeight: "1.15",
+                    textAlign: "left",
+                  },
+                }}
+              >
+                {t("landing:introduction_title")}
+              </Typography>
+              <Typography
+                variant="h2"
+                component="span"
+                sx={{
+                  [theme.breakpoints.up("md")]: {
+                    display: "inline-block",
+                    marginTop: theme.spacing(4),
+                    position: "relative",
+                  },
+                }}
+              >
+                {t("landing:introduction_subtitle")}
+                <StyledDivider />
+              </Typography>
+            </StyledIntroductionText>
+            <Box
+              display={{ xs: "none", md: "flex" }}
+              flexDirection="column"
+              width="100%"
+              mt={2}
             >
-              {t("landing:introduction_subtitle")}
-              <StyledDivider />
-            </Typography>
+              <MuiButton
+                onClick={scrollToMore}
+                variant="text"
+                size="medium"
+                sx={{
+                  color: theme.palette.common.white,
+                  background: "none",
+                  border: "none",
+                  textAlign: "center",
+                }}
+              >
+                Read more
```

Comment:
Don't forget to put this in i18n instead (the translation tool). If you search i18n you should find some examples.